src/features/telemetry/renderer/telemetry-white-list-for-functions.injectable.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+
+const telemetryWhiteListForFunctionsInjectable = getInjectable({
+  id: "telemetry-white-list-for-functions",
+  instantiate: () => ["some-placeholder-injectable-id"],
```

Comment:
Do we need this placeholder?

Reply:
It's there for documentation and inferred typing. I'd say no harm?

Reply:
Yeah, no harm.